collada_parser/include/collada_parser/collada_parser.h

```diff
@@ -38,16 +38,13 @@
 #define COLLADA_PARSER_COLLADA_PARSER_H
 
 #include <string>
-#include <map>
-#include <boost/function.hpp>
```

Comment:
Pre-release show no issues? Removing includes from public headers could break downstream code that is missing includes, but I'm fine with that if a pre-release build succeeds.

Reply:
Yeah, the prerelease was fine.  That doesn't guarantee that out-of-tree users aren't depending on it, but I think the somewhat small chance of breakage is worth it to cleanup the exported headers.